### PR TITLE
Less toasty

### LINKED ISSUE #409: Too Much Toast?
[This commit](https://github.com/umts/pvta-multiplatform/commit/6185b0ebb32818a7fab568121aabdbd613e475cd) added toasts on the Routes & Stops page when you favorite or unfavorite a route, but they don't hide themselves.  You could end up with an annoying number of them, like so:

toast

Is this worth fixing?

### PR DESCRIPTION
**Pull request by mboneil10**
Closes #409

### CODE REVIEW

**Comment by mboneil10** [comment]
less-toast

File: package.json [hunk 1]
```diff
@@ -50,12 +50,13 @@
...
     "@ionic/cli-build-ionic-angular": "^0.0.3",
-    "@ionic/cli-plugin-cordova": "^1.4.0",
-    "@ionic/cli-plugin-ionic-angular": "^1.3.1",
+    "@ionic/cli-plugin-cordova": "1.5.0",
+    "@ionic/cli-plugin-ionic-angular": "1.4.0",
     "@types/jasmine": "^2.5.41",
     "@types/node": "^7.0.8",
     "angular2-template-loader": "^0.6.2",
     "html-loader": "^0.4.5",
+    "ionic": "3.6.0",
```
Comment (akaplo) [inline package.json]: Let's leave all the `package.json` updates to the Ionic 3 PR.

**Review by akaplo — APPROVED** [review]
Hooray! Just see my comments on package.json, but those are easy changes so I'll approve now.

**Comment by mboneil10** [comment]
WHYYYYY

**Pull request merged**